README.md

```diff
@@ -109,24 +109,22 @@ Some tests need a redis deployed in `localhost:6379`. You can run it in Docker w
 docker run --rm -p 6379:6379 -it redis
 ```
 
+Some tests need a infinispan deployed in `localhost:11222`. You can run it in Docker with:
+```bash
+docker run --rm -p 11222:11222 -it -e USER=username -e PASS=password infinispan/server:11.0.9.Final
+```
+
 Then, run the tests:
 
 ```bash
-cargo test
+cargo test --all-features
```

Comment:
Did you mean to make these README changes as part of the dep update?
Any reason why you drop the Infinispan paragraph?

Reply:
`cargo test` already includes integration tests and requires Infinispan to be up and running. That is way I removed the paragraph.

I added `--all-features` because I want contributors to pass all the tests